### PR TITLE
typo: incorrect filename in google-firebase.mdx

### PR DESCRIPTION
**Pull request by Erik-McKelvey**
#### What kind of changes does this PR include?

- Minor content fixes (broken links, typos, etc.)

#### Description

- In most places the file and endpoint was called `signin.ts`. Changed the two places that inconsistency referenced `login` and `login.ts`.
- After review by TheOtterlord, changed `logout` to `signout` in two places as well

### CODE REVIEW

**Comment by kevinzunigacuellar** [comment]
LGTM! Thanks a lot for catching those @Erik-McKelvey

File: src/content/docs/en/guides/backend/google-firebase.mdx [hunk 1]
```diff
@@ -302,7 +302,7 @@ After creating server endpoints for authentication, your project directory shoul
   - pages
     - api
       - auth
-        - **login.ts**
+        - **signin.ts**
         - **logout.ts**
```
Comment (TheOtterlord) [inline src/content/docs/en/guides/backend/google-firebase.mdx:306]:
```suggestion
        - **signin.ts**
        - **signout.ts**
```

**Review by TheOtterlord — COMMENTED** [review]
Thanks @Erik-McKelvey! I can also see `signout`/`logout` is used inconsistently across the guide. There's two cases of `signout being used`. The first I've added a comment for, and the second is on line [230](https://github.com/withastro/docs/pull/4858/files#diff-29469be11da9d9215e54fd942c3ca458eefa7b8708ab45f442a3018c134c3bd1R247).

**Review by TheOtterlord — COMMENTED** [review]
Thanks a lot! Welcome to team docs :tada:

**Pull request merged**